content/kotlin/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'floor()'
+Description: 'The `floor()` method in Kotlin is used to round a floating-point to a nearest lowest integer value'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Methods'
+  - 'Floor'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`floor()`** function returns a rounded number down to the nearest integer.
+
+The result returns a value type of `float`.
+
+## Syntax
+
+```pseudo
+Math.floor(x:Double)
+```
+
+Where `x` is the number to be rounded down to the nearest integer.
+
+## Example
+
+```kotlin
+println(Math.floor(3.4))
+// Output: 3
+```
+
+The result will be a float value of `3` as the function rounds `3.4` down to `3`.
+
+## Codebyte Example
```

Comment:
kotlin is not currently supported by the Codecademy compiler so I would suggest removing this example